rfc9000.md

```diff
@@ -1829,7 +1829,7 @@ constraints conflict.
 
 Using 0-RTT depends on both client and server using protocol parameters that
 were negotiated from a previous connection.  To enable 0-RTT, endpoints store
-the values of the server transport parameters with any session tickets it
+the value of the server transport parameters with any session tickets it
```

Comment:
This seems inconsistent with the remainder of the paragraph and following, where we talk about "values" for the individual parameters.